### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by IchikawaYukko**
## What type of PR?

Translation

## What does this PR do?

Provide completed Japanese translation.

### Related issue(s)

None

## Prerequistes
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [*] In case of feature or enhancement: documentation updated accordingly
- [*] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/guide.html#changelog) entry file.

### CODE REVIEW

**Comment by IchikawaYukko** [comment]
bors try

**Comment by micw** [comment]
bors try

**Comment by Nebukadneza** [comment]
bors try

File: core/admin/mailu/translations/ja/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -3,17 +3,21 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: POEditor.com\n"
+"X-Generator: Poedit 1.5.7\n"
 "Project-Id-Version: Mailu\n"
 "Language: ja\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
```
Comment (Nebukadneza) [inline core/admin/mailu/translations/ja/LC_MESSAGES/messages.po]: Ah! It seems these 2 either need to be a valid date, or be removed! Didn’t know myself … m(_ _)m

**Comment by hoellen** [comment]
Please add the changelog entry as described in the [development guidelines](https://mailu.io/1.7/contributors/guide.html#changelog).

**Comment by IchikawaYukko** [comment]
Which file extention should be used for crier/newsfragments? .feature or .misc or another ?

**Comment by Nebukadneza** [comment]
I’d say `.feature`, since it is user-relevant (especially to Japanese users).

**Comment by IchikawaYukko** [comment]
Thanks Nebukadneza.

bors retry

**Comment by Nebukadneza** [comment]
Sorry for bors being so stubborn:

bors try

**Pull request merged**